<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
	background-color: #151515;
	color: #ffffff;

}

QLabel {
color: #ffffff;
}

QToolButton {
	color: #ffffff;
	background-color: #151515;
	border: 0px;
}</string>
   </property>
   <widget class="QLabel" name="real_time_hour">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>90</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="real_date">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
     </font>
    </property>
    <property name="text">
     <string>12.12.2019</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="utc_label">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>126</y>
      <width>51</width>
      <height>13</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>UTC.S : Using System Time</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>UTC.S</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sim_label">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>29</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Waiting Simulator</string>
    </property>
   </widget>
   <widget class="QLabel" name="sim_time_hour">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>7</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>01</string>
    </property>
   </widget>
   <widget class="QLabel" name="sim_date">
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>31</y>
      <width>81</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12.12.2019</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="left_status">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>126</y>
      <width>221</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="left_value">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>140</y>
      <width>221</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="live_button">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Live Sync: Disabled</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Live Sync</string>
    </property>
   </widget>
   <widget class="QToolButton" name="sync_button">
    <property name="geometry">
     <rect>
      <x>418</x>
      <y>60</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Sync Now</string>
    </property>
    <property name="text">
     <string>Sync Now</string>
    </property>
   </widget>
   <widget class="QToolButton" name="source_button">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>139</y>
      <width>41</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Source</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>6</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulator</string>
    </property>
   </widget>
   <widget class="QLabel" name="real_time_minute">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>20</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>90</pointsize>
     </font>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="real_time_seperator">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>20</y>
      <width>21</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>70</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sim_time_minute">
    <property name="geometry">
     <rect>
      <x>457</x>
      <y>7</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>30</string>
    </property>
   </widget>
   <widget class="QLabel" name="sim_time_seperator">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>10</y>
      <width>5</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>1</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: #ffffff</string>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="real_time_second">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>90</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="source_button_2">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>112</y>
      <width>41</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset</string>
    </property>
   </widget>
   <widget class="QLabel" name="right_status">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>121</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sim_time_second">
    <property name="geometry">
     <rect>
      <x>481</x>
      <y>11</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="right_status_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>135</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
